### PR TITLE
[CU-m16wt6] Add instructions to readme about adding slack bot

### PR DESCRIPTION
**Pull request by Zummek**
Plis czek maj englisz

### CODE REVIEW

**Comment by Zummek** [comment]
Task linked: [CU-m16wt6 Add instructions to readme about adding Slack bot](https://app.clickup.com/t/m16wt6)

File: server/slack-bot-manifest.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ display_information:
   background_color: "#1976d2"
```
Comment (eastemployeer) [inline server/slack-bot-manifest.yaml]: This file has nothing to do with adding readme to slackbot

File: README.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ It's the best tournament application for make fun in companies
...
+### Add Slack's bot
+
+First, create and install a slack bot based on prepared manifest file in Your workspace.
+
+1. Visit this page: https://api.slack.com/apps and click "Create New App"
+2. Select "From an app manifest" option, then pick your slack's workspace
+3. Copy [this manifest](slack-bot-manifest.yaml) file into input, next and create
+4. On app page click "Install to Workspace", You need to allow permissions for Your Slack's workspace.
+
+Then connect slack's bot with app server.
```
Comment (eastemployeer) [inline README.md]: Any details about this step?

**Review by eastemployeer — COMMENTED** [review]
This is just a shorted version of whole process. I think you should add more detailed informations.

**Review by eastemployeer — CHANGES_REQUESTED** [review]
Please add more details.

File: server/slack-bot-manifest.yaml (hunk 1 above)
Comment (Zummek) [reply server/slack-bot-manifest.yaml]: I checked slack bot setting/permissions for useless things few days ago. This file is important for creating bot, so I thought it was a good place to update that file

File: README.md (hunk 2 above)
Comment (Zummek) [reply README.md]: You have below two points about it

**Comment by Zummek** [comment]
@eastemployeer 
> Please add more details.

This is all what You need to do with slack bot to start application. What would You like to add? What are you missing?
Setting details (.env) should be described in another point / task.

**Review by eastemployeer — APPROVED** [review]

**Pull request closed without merging**